lib/AgrirouterClient/src/Protobuf/meson.build

```diff
@@ -0,0 +1,52 @@
+protoc_exe = find_program('protoc')
+
+pymodule = import('python3')
+python3 = pymodule.find_python()
```

Comment:
You support Meson >=0.46, which is the same version that `import('python')` was introduced. Shortly after, in Meson 0.48, the python3 one was deprecated after we concluded that it served no purpose anymore.